src/web3/strategy/strategies/controller.bdao.ts

```diff
@@ -37,54 +37,24 @@ export class BankTokenWeightedStrategy extends StrategyBaseController implements
     // modify: implement strategy here
     async strategy(
         ethAddress: string,
-        blockHeight: number | null,
+        blockHeights: BlockHeight[],
         evmService: EvmService,
         graphqlService: GraphqlService,
         logger: Logger,
         // tokenWhitelistService: TokenWhitelistMongoService,
     ) {
 
-        let equivalentBlock = null;
+        const equivalentBlock = blockHeights.find((blockHeight) => blockHeight.chain_id === '137').block;
```

Comment:
maybe need to rename to `polygonBlockHeight` as we plan to support other chains long term.

Maybe log as tech debt.